test/JDBC/expected/BABEL-3759.out

```diff
@@ -0,0 +1,66 @@
+EXEC sp_babelfish_configure 'enable_pg_hint', 'on'
```

Comment:
Please change the filename to `index-hint-vu-XXX` make this as a form of MVU test.